net-analyzer/gvm-libs/gvm-libs-22.10.0.ebuild

```
@@ -50,11 +50,16 @@ BDEPEND="
 	test? ( dev-libs/cgreen )
 "
 
-PATCHES=(
-	# Fix bug 925932
-	# See https://github.com/greenbone/gvm-libs/pull/811
-	"${FILESDIR}"/gvm-libs-22.8.0-linking-math-library.patch
-)
+pkg_setup() {
+	if tc-is-clang; then
+		CLANG_MAJOR_VERSION=$(clang-major-version);
+		if [[ ! -e "${ROOT}"/lib/clang/"${CLANG_MAJOR_VERSION}"/lib/linux/libclang_rt.profile-x86_64.a ]] ; then
```

This should use `has_version`, instead of checking for an arch specific file.

```suggestion
		if [[ ! has_version sys-libs/compiler-rt-sanitizers:${clang_major_version}[profile] ]] ; then
```